chainforge/react-server/src/store.js

```diff
@@ -74,27 +74,38 @@ export const initLLMProviders = [
     emoji: "🤖",
     model: "gpt-3.5-turbo",
     base_model: "gpt-3.5-turbo",
+    parent: "Open AI",
```

Comment:
I'm going to reformat this since putting the parent on the provider info means it'll save this info to a CF file, which is a waste of space ---this info is only used for the front-end UI.